package.json

```diff
@@ -147,7 +147,7 @@
     "electron-updater": "4.1.2",
     "history": "4.7.2",
     "hs-client": "https://github.com/handshake-org/hs-client.git",
-    "hsd": "https://github.com/kyokan/hsd/tarball/1e1292d",
+    "hsd": "https://github.com/handshake-org/hsd/tarball/a94ce87",
```

Comment:
hsd master branch should be considered unstable, I think it'd be more prudent to follow hsd releases either from github or npm (latest is v2.3.0) I think this is fine for now, we need a better hsd release process anyway with release candidates etc to ensure we don't release a bug. Whats interesting about you using this commit in particular is that now Bob will have the "presigned REVEAL" endpoint API: https://github.com/handshake-org/hsd/pull/529